Tests/tests/configure_and_test_integration_instances_test.py

```diff
@@ -1,4 +1,4 @@
-from Tests.configure_and_test_integration_instances import configure_modified_and_new_integrations
+from Tests.configure_and_test_integration_instances import XSOARBuild
```

Comment:
please add more build test cases, for both XSOAR and XSIAM. currently we don't have enough tests coverage to the flow.

Reply:
Added some tests.